tracer/test/snapshots/Security.AspNetCore2.__test=server.request.query_enableSecurity=True_enableBlocking=True_expectedStatusCode=200_url=_Health_-[$slice]=value.verified.txt

```diff
@@ -0,0 +1,192 @@
+[
```

Comment:
I think this snapshot file may also be unused now and can be deleted

Reply:
removed in b9a46c5f6